test/config/basic/basicConfig.test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const { stat } = require('fs');
+const { resolve, sep } = require('path');
+const { run, extractSummary } = require('../../utils/test-utils');
+
+describe('config flag test : Basic config file', () => {
```

Comment:
Each test (`it`) should not have dependencies from each other. Instead here, the second and the second have a dependency from the first one.

I'd suggest to use the `run` function for each `it`.

Reply:
Actually here, I am trying to use the same instance of the run method to test each part of its result.
they will use that same run instance in a single test suite. 
I think a single test suite should contain a single instance of the run method.
Is it fine or should I change it then !?  
@ematipico

Reply:
Or you wrap everything in one single `it` and there you do your assertions or you split in different `it` but you don't share same execution instance. You're using a mix which could case issue

Reply:
ohkay I will add everything in a single `it` .